scripts/lib/constants.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+USER_TYPES = {
+        'basic': 1,
+        'pro': 2,
+        'corp': 3,
+        }
+
+VIDEO_CATEGORY_IDS = {
+    'Film & Animation': 1,
+    'Autos & Vehicles': 2,
+    'Music': 10,
+    'Pets & Animals': 15,
+    'Sports': 17,
+    'Short Movies': 18,
+    'Travel & Events': 19,
+    'Gaming': 20,
+    'Videoblogging': 21,
+    'People & Blogs': 22,
+    'Comedy': 34,
+    'Entertainment': 24,
+    'News & Politics': 25,
+    'Howto & Style': 26,
+    'Education': 27,
+    'Science & Technology': 28,
+    'Nonprofits & Activism': 29,
+    'Movies': 30,
+    'Anime/Animation': 31,
+    'Action/Adventure': 32,
+    'Classics': 33,
+    'Documentary': 35,
+    'Drama': 36,
+    'Family': 37,
+    'Foreign': 38,
+    'Horror': 39,
+    'Sci-Fi/Fantasy': 40,
+    'Thriller': 41,
+    'Shorts': 42,
+    'Shows': 43,
+    'Trailers': 44,
+}
+
+# 'Uploads from Zoom' is an unlisted playlist, 
+# so it's ID is stored in an environment variable.
+PLAYLIST_IDS = {
+    'Data Together': 'PLtsP3g9LafVul1gCctMYGm9sz5FUWr5bu',
+    'Website Monitoring': 'PLtsP3g9LafVtj6IOMk05aOh-DdpuqWhue',
+    'Community Calls': 'PLtsP3g9LafVsaa18lQaPXzxJU7wIcPB1O',
+    'Uploads from Zoom': os.environ['DEFAULT_YOUTUBE_PLAYLIST']
```

Comment:
This feels really brittle to me. Not sure what a better pattern would be...

Reply:
Yeah, I would probably not put this here. My tendency would be to either:

- Just let `add_video_to_playlist()` do its thing and use the API to look up the playlist (it’s pretty cheap) rather than needing the ID.

- Look up the env var for this in the main script, and only add to the default if present. e.g. in `upload_zoom_recordings.py`, something like:

    ```py
    default_playlist = os.environ.get('DEFAULT_YOUTUBE_PLAYLIST')

    # ...and down where you upload the video...
    video_id = upload_video(blah, blah, blah)
    if default_playlist:
        print('  Adding to main playlist: Uploads from Zoom')
        add_video_to_playlist(youtube, video_id, title=default_playlist,
                              privacy='unlisted')
        # Or if the env var is an ID instead of a name:
        print('  Adding to main playlist: Uploads from Zoom')
        add_video_to_existing_playlist(youtube, video_id, default_playlist)
    ```